lib/venue.rb

```diff
@@ -1,2 +1,38 @@
 class Venue
+
+  attr_accessor :name, :capacity, :patrons
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def add_patron(patron_name)
+    @patrons << patron_name
+  end
+
+  def yell_at_patrons
+    yelled_patrons = []
+    yelled_patrons << @patrons.map(&:upcase)
```

Comment:
Since you are using a `.map` here, you would not need to create an accumulator. Map will modify the original array. That is why you had to `.flatten` it. So it would just look like:

```
@patrons.map do |patron|
   patron.upcase
end
```